Build fails with Gcc >= 8 due to implicit conversion

When building with gcc 8 and above, I have the following error:

```bash
/home/travis/build/xtensor-stack/xtensor/build/test/googletest-src/googletest/src/gtest.cc:2112:72: error: conversion to ‘long unsigned int’ from ‘int’ may change the sign of the result [-Werror=sign-conversion]
 std::vector<std::string> ArrayAsVector(const char* const (&array)[kSize]) {
```

--- a/googletest/src/gtest.cc
+++ b/googletest/src/gtest.cc
@@ -2108,7 +2108,7 @@ static const char* const kReservedOutputTestCaseAttributes[] = {
     "classname",   "name", "status", "time",   "type_param",
     "value_param", "file", "line",   "result", "timestamp"};
 
-template <int kSize>
+template <size_t kSize>
 std::vector<std::string> ArrayAsVector(const char* const (&array)[kSize]) {
   return std::vector<std::string>(array, array + kSize);
 }

